.github/autolabeler.yml

```diff
@@ -0,0 +1,83 @@
+
+# The following paragraph was taken from the Apache Spark project.
+#   see: https://github.com/apache/spark/blob/master/.github/autolabeler.yml
+# Bot page: https://github.com/apps/probot-autolabeler
+# The matching patterns follow the .gitignore spec.
+# See: https://git-scm.com/docs/gitignore#_pattern_format
+# Also, note that the plugin uses 'ignore' package. See also
+# https://github.com/kaelzhang/node-ignore
+
+INFRA:
+  - ".baseline/"
+  - ".github/"
+  - "/dev/"
+  - ".asf.yaml"
+  - ".gitattributes"
+  - ".gitignore"
+  - "baseline.gradle"
+  - "deploy.gradle"
+  - "jitpack.yml"
+  - "site/mkdocs.yml"
+  - "travis.yml"
+BUILD:
+  - "/dev/"
+  - "/project/"
+  - "/assembly/"
+  - "*pom.xml"
+  - "/bin/docker-image-tool.sh"
+  - "/bin/find-spark-home*"
+  - "gradle/"
+  - "**.gradle"
+BASH:
+  - "dev/check-license"
+  - "**.sh"
+DOCS:
+  - "site/"
+  - "!/site/mkdocs.yml"
+  - "**CHANGELOG.md"
+  - "**LICENSE"
+  - "**NOTICE"
```

Comment:
I think that LICENSE and NOTICE changes should be tagged "license" instead.

Reply:
I've made a LICENSE tag that will match any files (or directories technically) named LICENSE or NOTICE.

```yaml
LICENSE:
  - "**LICENSE"
  - "**NOTICE"
```